omise/payment_methods/alipay.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . 'omise/payment_methods/_offsite.php';
+}
+
+class OmisePaymentMethod_Alipay extends OmiseOffsitePaymentMethod
+{
+
+    const
+        NAME = 'Alipay',
+        PAYMENT_OPTION_NAME = 'omise-alipay-payment',
+        DEFAULT_TITLE = 'Alipay',
+        TEMPLATE = 'alipay_payment',
+        CONTROLLER = 'alipaypayment'
```

Comment:
`const CONTROLLER` need to remove this if never used.

Reply:
Already removed in a later PR